spec/policies/user_policy_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+RSpec.describe UserPolicy do
```

Comment:
I think this way is clearer to read but its a good point about adding new actions.
Can add at a later date as the current tests cover all the current actions